internals/models.py

```diff
@@ -732,12 +747,12 @@ def get_all(self, limit=None, order='-updated', filterby=None,
     feature_list = ramcache.get(KEY)
 
     if feature_list is None or update_cache:
-      query = Feature.all().order(order) #.order('name')
-      query.filter('deleted =', False)
+      query = Feature.query().order(Feature.order) #.order('name')
+      query.filter(Feature.deleted == False)
 
       # TODO(ericbidelman): Support more than one filter.
       if filterby:
-        query.filter(filterby[0], filterby[1])
+        query.filter(Feature.filterby[0] == filterby[1])
```

Comment:
The method signature of `query.filter()` has changed. Just to verify, I hope I have migrated this line correctly.

Reply:
I don't think that that can work because there is no 'filterby' field defined, so Feature.filterby will be an error.  The part before the operator needs to be a field defined in the entity class.

There is only one caller that uses the filterby= keyword parameter.  That is in featurelist.py and it only ever passes in ('category =', k).   So, you could take out the filterby= keyword and instead implement a category= keyword that does query.filter(Feature.category == category).

Reply:
Thanks. I have committed this change.